content/company/strategy/code-graph/code-intelligence/index.md

```diff
@@ -65,13 +66,13 @@ Recent key learnings:
 - Requests for adding precise support for more languages (often including the addition of cross-repository and cross-dependency navigation features).
 - Current manual setup is not straightforward, customers run into issues while setting up LSIF for their repositories. This varies per indexer maturity and language ecosystem complexity.
 - There have been reports about performance and scaling issues when indexing large monorepos.
-- Requests feature that helps visualize the code graph and it’s dependencies.
+- Requests feature that helps visualize the code graph and its dependencies.
 
 ### Competitive landscape
 
-- **Code hosts:** the most important code hosts have navigation features, mostly powered by imprecise code intelligence. They had good out of the box support, but will never be as accurate as our precise code intelligence unless they implement an LSIF solution themselves. So far this hasn’t happened, but is a potential risk.
-- **IDEs:** code navigation is a core part of a developer’s workflow and most IDEs have advanced navigation features we cannot avoid competing with. Most developers expect Sourcegraph’s code navigation to have feature parity with their favorite IDE.
-- **Other navigation tools:** these are usually focused on one or only a few languages, none of them have reached wide usage or compete directly with our value proposition.
+- **Code hosts:** The most relevant code hosts have navigation features, mostly powered by imprecise code intelligence. They have good out-of-the-box support, but their chosen technical path prioritizes wider language breath sacrificing accuracy. Our precise code intelligence approach focuses on having 100% accuracy by implementing LSIF indexers. So far none of our competitors have mimicked the approach, but as precise code navigation gains traction this is a potential risk.
```

Comment:
If the most relevant is not a long list, we could explicitly list them here.

Reply:
You could also mention we are staying ahead of that risk by ensuring we continue to innovate and build hard to copy features (assuming that's true).

Reply:
```suggestion
- **Code hosts:** Both GitHub and GitLab have navigation features, mostly powered by imprecise code intelligence. They have good out-of-the-box support, but their chosen technical path prioritizes wider language breath sacrificing accuracy. Our precise code intelligence approach focuses on having 100% accuracy by implementing LSIF indexers. So far none of our competitors have mimicked the approach, but as precise code navigation gains traction this is a potential risk.
```